pinot-core/src/main/java/org/apache/pinot/core/common/ObjectSerDeUtils.java

```diff
@@ -213,6 +216,8 @@ public static ObjectType getObjectType(Object value) {
         return ObjectType.VarianceTuple;
       } else if (value instanceof PinotFourthMoment) {
         return ObjectType.PinotFourthMoment;
+      } else if (value instanceof org.apache.datasketches.tuple.Sketch) {
+        return ObjectType.IntegerTupleSketch;
```

Comment:
Is this a safe assumption?  Is it also necessary to inspect the summary type to verify integer?

Reply:
right now it is, but to add other types of tuple Sketch we'd need to add wrapper types